### DISCUSSION TITLE
[Help] Bucle on subscriptions

### DISCUSSION THREAD

**Discussion by jeromflo** [ReactiveX/rxjs#6938] [Q&A]
### Describe the bug

Dear core team. 
I am a angular developer , actually I have pleassure for rxjs and   ngrx and the world of subscription. Actually I have  a problem, a infernal bucle on subscription.
Where  I subscribe to changes on form and change this with a events. I will explain this. I have my store and my reactive form. 
On changes in my  store, I update the data on my form, and changes on my form I update my changes on my store. It's sure is bad practice on the world of suscriptions (please verify this), but I update my component dinamically with changes and have temporal persistance with the store. 
For break this bucle only I thinks 2 forms: 

1. Put a button where call a method where update store, and break this bucle, this don't like because  I wouldn't like for ussability on web, concretly on this form. This way I think about control variables and the pipe first(), where wont subscribe the changes on form. 
2. Same method or way already implemented, where delete this problem. 

I looked from internet and didn't see things. 

Please respond me and excuse me the lost time.

### Expected behavior

Somethings of knowledge from the better way of fixed this problem.
For example, update the store, and store won't update the form.

### Reproduction code

```typescript
constructor(private store:Store<{form:{name:string,age:number}}>private fb:FormBuilder){
this.persomForm=this.fb.group({
name:[''],
age:['']
})
this.persomForm.valueChanges.subscribe(el=>{
let name=el.name;
let age=el.age;
 this.store.dispatch(actions.setName({value:name}))
 this.store.dispatch(actions.setAge({value:age}))
});
this.store.subscribe((el:{name:string,age:number})=>{
this.persomForm.patchValue(el);
});
}
```

### Version

RXJS: 6.6.0   NGRX:1 13.0.2 Angular 12.2.16 node14.17.6 npm 6.14.15

**Comment by kwonoj** [comment]
Moved to q&a.

**Reply by jeromflo** [reply]
Thanks
